1.architectures/5.sagemaker-hyperpod/LifecycleScripts/base-config/config.py

```diff
@@ -27,6 +27,17 @@ class Config:
     # Set true to install quality-of-live improvements
     enable_initsmhp = False
 
+    # Set true if you want to use mountpoint for s3 on cluster nodes. 
+    # If enabled, a systemctl mount-s3.service file will be writen that will mount at /mnt/<BucketName>.
+    # requires s3 permissions to be added to cluster execution role. 
+    enable_mount_s3 = False
+
+    data_bucket = "" # when enable_mount_s3 = True, replace with your actual data bucket name in quotes, ie. "my-data-bucket"
```

Comment:
call this `s3_bucket`:
```suggestion
    s3_bucket = "" # when enable_mount_s3 = True, replace with your actual s3 bucket name in quotes, ie. "my-s3-bucket"
```